publications/papers/optimal-sizing-paper/acks.tex

```diff
@@ -0,0 +1,6 @@
+\section{Acknowledgments}
```

Comment:
I hope this wasn't the acknowledgements file you included in your actual optimal sizing ANS summary submission... or is this a journal article version?... What am I looking at here?

Reply:
No. This is a draft, that should not have been included in the PR. It's literally just the template (and possibly some notes I was making to myself in other files).